solidity/migrations/2_deploy_contracts.js

```diff
@@ -59,6 +60,33 @@ module.exports = async function(deployer, network) {
     KeepRegistry.address,
     initializationPeriod
   );
+
+  let oldStakingContractAddress;
+  if (network === 'ropsten') {
+    // 1.3.0-rc.0 TokenStaking contract address
+    oldStakingContractAddress = '0x8117632eC1D514550b3880Bc68F9AC1A76c9C67B';
+  } else if (network === 'mainnet') {
```

Comment:
not super strong on this one, but maybe `mainnet` and then `ropsten` in the if statement?

Reply:
Changed in https://github.com/keep-network/keep-core/pull/2017/commits/54a6bdcb9325fcbb8ccabe82ef1dd0dc8cbcafdb